### PR TITLE
Add show/hide password button on login page

### PR DESCRIPTION
**Pull request by Bubu**

### CODE REVIEW

File: resources/qml/dialogs/LeaveRoomDialog.qml [hunk 1]
```diff
@@ -13,7 +13,7 @@ P.MessageDialog {
     property string reason: ""
 
     title: qsTr("Leave room")
-    text: qsTr("Are you sure you want to leave?")
+    text: qsTr("Are you sure you want to leave %1?").arg(Rooms.getRoomPreviewById(roomId).roomName)
```
Comment (Bubu) [inline resources/qml/dialogs/LeaveRoomDialog.qml:16]: Lol, I guess I still had that somewhere in my tree.

File: resources/qml/pages/LoginPage.qml [hunk 2]
```diff
@@ -82,15 +82,39 @@ Item {
...
+                    Keys.forwardTo: [pwBtn, ssoRepeater]
+                }
+
+                ImageButton {
+                    id: showPwButton
+                    Layout.preferredWidth: 30
+                    Layout.preferredHeight: 30
+                    visible: login.passwordSupported
+                    Layout.alignment: Qt.AlignBottom
+                    image: ":/icons/icons/ui/eye-show.svg"
```
Comment (deepbluev7) [inline resources/qml/pages/LoginPage.qml:103]:
```suggestion
                    image: passwordLabel.echoMode = TextInput.Password ? ":/icons/icons/ui/eye-show.svg" : ":/icons/icons/ui/eye-hide.svg"
```

File: resources/qml/pages/LoginPage.qml [hunk 3]
```diff
@@ -82,15 +82,39 @@ Item {
...
+                        if (passwordLabel.echoMode === TextInput.Normal) {
+                            passwordLabel.echoMode = TextInput.Password
+                            showPwButton.image = ":/icons/icons/ui/eye-show.svg"
+                        }
+                        else {
+                            passwordLabel.echoMode = TextInput.Normal
+                            showPwButton.image = ":/icons/icons/ui/eye-hide.svg"
+                        }
+                    }
+                    onClicked: showPwButton.togglePasswordVisibility()
```
Comment (deepbluev7) [inline resources/qml/pages/LoginPage.qml:116]:
```suggestion
                    onClicked: {
                        if (passwordLabel.echoMode === TextInput.Normal) {
                            passwordLabel.echoMode = TextInput.Password
                        }
                        else {
                            passwordLabel.echoMode = TextInput.Normal
                        }
                    }
```

No real need for a separate function here :)

**Review by deepbluev7 — CHANGES_REQUESTED** [review]
Looks good, a few small suggestions

File: resources/qml/pages/LoginPage.qml (hunk 2 above)
Comment (Bubu) [reply resources/qml/pages/LoginPage.qml]: That needed a `===` instead of `=` in the condition.

**Comment by Bubu** [comment]
Thanks for the review, suggestions applied.

**Comment by deepbluev7** [comment]
Thanks, seems to work nicely!

**Pull request merged**